ts/features/fims/history/navigation/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { createStackNavigator } from "@react-navigation/stack";
+import * as React from "react";
+import { FimsHistoryScreen } from "../screens/HistoryScreen";
+
+export const FIMS_HISTORY_ROUTES = {
+  MAIN: "FIMS_HISTORY_MAIN",
+  HISTORY: "FIMS_HISTORY_HISTORY"
+} as const;
+
+export type FimsHistoryParamsList = {
+  [FIMS_HISTORY_ROUTES.MAIN]: undefined;
+  [FIMS_HISTORY_ROUTES.HISTORY]: undefined;
+};
+
+const Stack = createStackNavigator<FimsHistoryParamsList>();
+
+export const FimsHistoryNavigator = () => (
```

Comment:
What do you think of merging this navigator with the FIMS_SSO one, since both of them only have one screen each?

Reply:
well you're right, on it!